lib/Controller/SignalingController.php

```diff
@@ -150,6 +150,13 @@ public function pullMessages($token) {
 		$seconds = 30;
 		$sessionId = '';
 
+		try {
+			$sessionId = $this->session->getSessionForRoom($token);
```

Comment:
You could return immediately if `$sessionId` is `null` here (similar as below).

Reply:
done